client/app/components/TimeAgo.jsx

```diff
@@ -0,0 +1,91 @@
+import moment from 'moment';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+
+const interactiveComponents = new Set();
+
+function updateInteractiveComponents() {
+  interactiveComponents.forEach(component => component.forceUpdate());
+  setTimeout(updateInteractiveComponents, 3000);
```

Comment:
The value will update at most every minute. Why check every 3 seconds?

Reply:
It's not related to how often value will be updated. It's for updating an amount of time between value and now. For example, when you execute query - it's `executed_at` will be updated at once, but component should periodically show new value: `1 minute ago`, then `2 minutes ago` and so on. `angular-moment` has a progressive refresh interval, depending on time difference (in first seconds it's 1000ms, after a minute it's 1min, after an hour - 60min and so on).

Reply:
Increased polling interval to 15 seconds.

Reply:
I meant how often we need to update the value on screen... :)